template/src/models/models.js

```diff
@@ -24,7 +24,7 @@ export default async function schemaRender({ asyncapi }) {
       }
     ]
   });
-  const generatedModels = await typescriptGenerator.generateCompleteModels(asyncapi, {moduleSystem: 'ESM'});
+  const generatedModels = await typescriptGenerator.generateCompleteModels(JSON.parse(originalAsyncAPI), {moduleSystem: 'ESM'});

```

Comment:
`originalAsyncAPI` can be defined as YAML, so `JSON.parse(originalAsyncAPI)` won't work 😅

Reply:
Right... 🤔

Reply:
but... You can put it as JSON or YAML because `parser` handles YAML and JSON, don't worry :)

Reply:
Aaa ok, I see this.

Reply:
Definitely a possibility 👍 Feel free to raise it as a feature 😄

Reply:
Could you do this for me? 😆

Reply:
No, it's your feature suggestion 😆